opm/output/eclipse/CreateInteHead.cpp

```diff
@@ -273,6 +361,9 @@ namespace {
     {
         const auto& wd = rspec.wellDimensions();
 
+        // for LGR, grpsz, and ngrp are LGR property
+        // and maxWellsPerGroup, maxGroupsInField are always global properties.
+        // therefore, this does not need to be changed
```

Comment:
I'd rather prefer that this comment be removed entirely, but if you really want it, then it should say something along the lines of

> Maximum number of wells per group and maximum number of groups in the field apply to all grids.